docs/usage/storage.rst

```diff
@@ -0,0 +1,48 @@
+Storage
+=======
+
+.. todo::
+   Intro sentence.
+
+Volumes
+-------
+
+Any workload needs different volumes with different storage capacities
+to store its components. Following the total storage capacity needed,
+establish a number of volumes and their capacities for each node, as
+exemplified below:
+
+.. code-block:: yaml
+
+    metalk8s_lvm_lvs_vg_metalk8s: ['/dev/vdb']
+        lv01:
+            size: 52G
+        lv02:
+            size: 52G
+        lv03:
+            size: 52G
+        lv04:
+            size: 11G
+        lv05:
+            size: 11G
+        lv06:
+            size: 11G
+        lv07:
+            size: 5G
+        lv08:
+            size: 5G
+
+Resize lv
+---------
+
+Volumes' capacities can be resized (one or several at once).
+Change the volume size value to a higher one and run:
+
+.. code::
+
+  ansible-playbook -i 'your inventory' -t storage playbooks/deploy.yml
+
+Both file system and logical volumes are automatically resized.
```

Comment:
Tested yesterday, This doesn't work. I had to run 
```
resize2fs /dev/mapper/vg_metalk8s-lv01
```
To have my FS resized. But hold on with the doc, we'll try to have a PR that fix the behaviour and make it work like the docs states.

Reply:
See #391